src/playlist-selectors.js

```diff
@@ -242,3 +242,39 @@ export const movingAverageBandwidthSelector = function(decay) {
                           parseInt(safeGetComputedStyle(this.tech_.el(), 'height'), 10));
   };
 };
+
+export const minRebufferingSelector = function(master,
```

Comment:
It's not grabbing just the max bandwidth that wont have rebuffering. Its returning the playlist that will provide the minimum amount of time rebuffering. If there is a playlist where the amount of time rebuffering is 0, then it will be the max-bandwidth that can do it, but if none can bring that to 0, it will pick whichever has the least impact on rebuffering times